code_samples/discounts/src/Command/ManageDiscountsCommand.php

```diff
@@ -78,7 +79,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 new IsInRegions(['germany', 'france']),
                 new IsProductInArray(['product-1', 'product-2']),
                 new IsInCurrency('EUR'),
-                new IsValidDiscountCode($discountCode->getCode(), $discountCode->getUsedLimit()),
+                new IsValidDiscountCode($discountCode->getCode(), $discountCode->getGlobalLimit(), $discountCode->getUsedLimit()),
```

Comment:
Not a biggie, but I always think this greatly increases readability:
```suggestion
                new IsValidDiscountCode(
                    $discountCode->getCode(),
                    $discountCode->getGlobalLimit(),
                    $discountCode->getUsedLimit()
                ),
```